crates/lib/src/executable.rs

```diff
@@ -42,7 +43,9 @@ use quil_rs::program::ProgramError;
 /// #[tokio::main]
 /// async fn main() {
 ///     use std::num::NonZeroU16;
-///     let mut result = Executable::from_quil(PROGRAM).with_client(Qcs::default()).with_shots(NonZeroU16::new(4).unwrap()).execute_on_qvm().await.unwrap();
+///     use qcs::qvm;
+///     let qvm_client = qvm::http::HttpClient::new(Qcs::load().await.get_config().qvm_url().to_string());
```

Comment:
If this is the way we want people to initialize the client, should we offer a `from_qcs` constructor?